pkg/capr/installer/installer.go

```diff
@@ -192,8 +212,9 @@ $env:CSI_PROXY_URL = "%s"
 $env:CSI_PROXY_VERSION = "%s"
 $env:CSI_PROXY_KUBELET_PATH = "C:%s/bin/kubelet.exe"
 $env:STRICT_VERIFY = "%s"
+$env:CATTLE_ROLE_NONE="true"
```

Comment:
Can you confirm that it is safe to have  `$env:CATTLE_ROLE_NONE="true` without checking if the token is empty or not?

Reply:
Yes, this is fine. This was previously set [within the `token` variable along with the actual token](https://github.com/rancher/rancher/blob/17221038da5bfbc17e5cd728c50925b599941ee8/pkg/capr/installer/installer.go#L163) (for some reason), and there isn't a case where it will never not be equal to `true`. We even have [a unit test](https://github.com/rancher/rancher/blob/17221038da5bfbc17e5cd728c50925b599941ee8/pkg/capr/installer/installer_test.go#L34) to ensure that it has this value.

Reply:
Good point, to be honest I'm not sure if there is a scenario where rancher delivers this script without a token being provided, without it the node wouldn't be able to contact the Rancher API. But, just to be save I've only set it when we do have the token value